components/com_fields/layouts/field/render.php

```diff
@@ -28,14 +30,24 @@
 	return;
 }
 
+$canEdit = $item->params->get('access-edit');
+$dataAttributes = '';
+$inlineEditClass = '';
+
+if ($canEdit && $field->type == 'text')
+{
+	$dataAttributes = HTMLHelper::_('convertToDataAttributes', 'com_fields', 'field', [ 'item_id' => $item->id, 'field_id' => $field->id ]);
```

Comment:
You can spread all the arguments over multiple lines like this:
```
$dataAttributes = HTMLHelper::_(
   'convertToDataAttributes',
   'com_fields',
   and so on...
```
No need for spaces around the brackets.